### ISSUE TITLE
DateFnsUtils not exported from @material-ui-pickers/adapter/date-fns

### ISSUE THREAD

**Issue by callmeberzerker** [mui-org/material-ui-pickers#1508]
## Expected behavior

There is `DateFnsUtils` exported from `@material-ui-pickers/adapter/date-fns`

## Actual behavior

In v.4.0.0.alpha.2 release notes it says DateFns adapter is exposed by the lib but it doesn't.

```
-import DateFnsUtils from '@date-io/date-fns'
+import DateFnsUtils from '@material-ui-pickers/adapter/date-fns'

<MuiPickersUtilsProvider utils={DateFnsUtils}> 
```

**Comment by dmtrKovalenko** [comment]
@material-ui/pickers/adapter/date-fns

**Comment by stefensuhat** [comment]
> @material-ui/pickers/adapter/date-fns

`unable to resolve path module`. I've tried reinstall the package, and the adapter folder still not there.

**Comment by dmtrKovalenko** [comment]
Fixed in v4.0.0-alpha.3
Sorry for that 😢
